src/microbe_stage/PatchMapNameGenerator.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Godot;
+
+public class PatchMapNameGenerator : IRegistryType
+{
+    private int syllabelsLowerLimit;
+    private int syllabelsHigherLimit;
+    private string continentName = null!;
+    private string patchName = null!;
+    [JsonRequired]
+    private List<string> syllabels = null!;
+    [JsonRequired]
+    private List<string> sufixes = null!;
+    public string InternalName{ get; set; } = null!;
+    public PatchMapNameGenerator(int lowerLimit, int higherLimit)
+    {
+        syllabelsLowerLimit = lowerLimit;
+        syllabelsHigherLimit = higherLimit;
```

Comment:
```suggestion
        syllablesLowerLimit = lowerLimit;
        syllablesHigherLimit = higherLimit;
```

Reply:
as you can probably guess i really thought this was the correct form, i kept spelling it in my head but coudlnt figure it out